CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 1.0.0
+This release goes from using Cognite API version 0.5/0.6 to using [Cognite API v1](https://docs.cognite.com/api/v1/). 
+All reads from, and writes to, CDF now uses the [Cognite Scala SDK](https://github.com/cognitedata/cognite-sdk-scala).
```

Comment:
it uses, they use

Reply:
True! :man_facepalming: Good catch, please keep being pedantic about grammar.

Reply:
I think that goes without saying 😉